### DISCUSSION TITLE
For how many of you does any url with the word crosh in it automatically close on your chromebook?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1188] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
^
Mine does lol.

**Comment by Asriel249** [comment]
yeah, is there any way to fix that?

**Reply by solanuscasey** [reply]
no, it is set in the policies for your chromebook
chrome://policy/
If not blocked specifically it may be possible to bypass using the link preview flag.

**Comment by schoolexploitkid** [comment]
hm might try to bypass by spamming

**Reply by schoolexploitkid** [reply]
idk tho opening 400 crosh crashed chrome

**Reply by solanuscasey** [reply]
if blocked by an app then spamming will work in some cases. if blocked by policy it will not work.

**Reply by solanuscasey** [reply]
> if blocked by an app<
in this case you can use the preview link flag to get around it
